src/vorta/borg/create.py

```diff
@@ -45,6 +45,7 @@ def started_event(self):
         self.app.backup_progress_event.emit(self.tr('Backup started.'))
 
     def finished_event(self, result):
+        self.app.reply(f"created {result['data']['archive']['name']}")
```

Comment:
The tests are failing because even during a normal backup I try to call `self.app.reply()`, which I should not. @real-yfprojects can you help me understand how I can check if the backup creation was triggered from the command line or normally from the GUI, so I can call `reply` optionally here?

Reply:
You can connect a handle to `BorgJob.result` when triggered from the cmd command.